<p>I recently installed a Postfix/Dovecot configuration that caused a Python script to send (or attempt to) outbound emails (name@anotherdomain.com) to local mailboxes (name@mydomain.com). <strong>Python3</strong> is using <strong>smtplib</strong> module with the following function to connect and send: </p>  <pre><code>smtplib.SMTP(host='127.0.0.1', port=25,  local_hostname="mydomain.com") </code></pre>  <p>See my mail.log for example:</p>  <p>Outbound log    </p>  <pre><code>Sep 21 22:53:03 mydomain postfix/smtpd[1870]: connect from localhost[127.0.0.1] Sep 21 22:53:03 mydomain postfix/smtpd[1870]: 7C98FE0F6A: client=localhost[127.0.0.1] Sep 21 22:53:03 mydomain postfix/cleanup[1873]: 7C98FE0F6A: message-id=&lt;20150921205303.7C98FE0F6A@mydomain.com&gt; Sep 21 22:53:03 mydomain postfix/qmgr[19853]: 7C98FE0F6A: from=&lt;sendingname@mydomain.com&gt;, size=374031, nrcpt=1 (queue active) </code></pre>  <p>Inbound log</p>  <pre><code>Sep 21 22:53:03 mydomain dovecot: lmtp(1879): Connect from local Sep 21 22:53:03 mydomain postfix/smtpd[1870]: disconnect from localhost[127.0.0.1] Sep 21 22:53:03 mydomain postfix/lmtp[1875]: 7C98FE0F6A: to=&lt;recevingname@mydomain.com&gt;, orig_to=&lt;recevingname&gt;, relay=mydomain.com[private/dovecot-lmtp], delay=0.09, delays=0.07/0/0/0.01, dsn=5.1.1, status=bounced (host mydomain.com[private/dovecot-lmtp] said: 550 5.1.1 &lt;recevingname@mydomain.com&gt; User doesn't exist: recevingname@mydomain.com (in reply to RCPT TO command)) Sep 21 22:53:03 mydomain dovecot: lmtp(1879): Disconnect from local: Successful quitSuccessful quit Sep 21 22:53:03 mydomain postfix/qmgr[19853]: 91F95E0F6C: removed </code></pre>  <p>Is there a way I can force Python/smtplib to send to external mailboxes. I know that the Postfix/Dovecot configuration is enabled to do so because a PHP script hosted on same server using <strong>phpmailer</strong> on the smtp port (25) is doing so without any hassle. </p>